cypress/e2e/clue/branch/student_tests/simulator_tile_spec.js

```diff
@@ -90,6 +90,23 @@ context.skip('Simulator Tile with Brainwaves Gripper Simulation', function () {
   });
 });
 
+context.skip('Simulator Tile with Brain Simulation', function() {
```

Comment:
Should this be skipped?

Reply:
All tests in this spec file are skipped at the moment - the loading times for these problem documents are quite high causing Cypress to timeout almost always. I've included this test in here and have kept them skipped for now following the general pattern of the rest of the tests in this file for when we're able to improve the loading times and unskip them.